frontend/awx/resources/templates/WorkflowVisualizer/hooks/useWorkflowVisualizerToolbarActions.tsx

```diff
@@ -43,6 +43,11 @@ export function useWorkflowVisualizerToolbarActions() {
         </ToolbarItem>
       </ToolbarGroup>
       <ToolbarGroup align={{ default: 'alignRight' }}>
+        <ToolbarItem style={{ paddingTop: '5px' }}>
+          <div data-cy="workflow-visualizer-toolbar-total-nodes">
+            {t('Total nodes')} <Badge>{nodes?.length || 0}</Badge>
```

Comment:
https://www.sketch.com/s/46cfe983-742c-4843-a462-6bd7bc04d8e1/a/27GybgG

In the mockups the "Total nodes" badge should be gray. PatternFly's Badge component has a prop called `isRead` that can easily accomplish this. https://www.patternfly.org/components/badge#read